src/components/Home/Home.module.scss

```diff
@@ -0,0 +1,30 @@
+.HomeView {
+  width: 50%;
+  background-color: white;
+  border-radius: 2em;
+  margin: 2em 0 5em 5em;
+  padding: 0 2em 0 2em;
+}
+
+.btnBar {
+  display: flex;
+  width: 75%;
+  justify-content: space-evenly;
+}
+
+.answerBtn {
+  cursor: pointer;
+  height: 2em;
+  border: 2px solid grey;
+  border-top: none;
+  border-left: none;
+  background: none;
+  outline: none;
+  border-radius: 1em;
+  box-shadow: 1px 1px 3px rgba(0, 0, 0, 0.452);
+  transition: all .15s ease-in-out;
+
+  &:hover {
+    background: rgba(117, 221, 255, 0.342);
+  }
```

Comment:
I think we need to create a shared stylesheet with some mixins for these views, and buttons.